lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -866,23 +920,19 @@ namespace Js
         }
 
         Assert(numberFormatter);
-        // REVIEW (doilij): AutoPtr will call delete on IPlatformAgnosticResource and complain of non-virtual dtor. There are no IfFailThrowHr so is this still necessary?
-        // TODO (doilij): If necessary, introduce an PlatformAgnosticResourceAutoPtr that calls Cleanup() instead of delete on the pointer.
-        // AutoPtr<IPlatformAgnosticResource> numberFormatterGuard(numberFormatter);
 
         // TODO (doilij): Render signed zero.
-
+        // TODO(jahorto): isDecimalPointAlwaysDisplayed seems like a Windows-only construct
         bool isDecimalPointAlwaysDisplayed = false;
-        bool useGrouping = true;
-
         if (GetTypedPropertyBuiltInFrom(options, __isDecimalPointAlwaysDisplayed, JavascriptBoolean))
         {
             isDecimalPointAlwaysDisplayed = JavascriptBoolean::FromVar(propertyValue)->GetValue();
         }
 
         if (GetTypedPropertyBuiltInFrom(options, __useGrouping, JavascriptBoolean))
         {
-            useGrouping = JavascriptBoolean::FromVar(propertyValue)->GetValue();
+            bool useGrouping = JavascriptBoolean::FromVar(propertyValue)->GetValue();
```

Comment:
Oh hm... useGrouping was not read at all previously? I'm surprised we didn't get a compiler warning for unused var. Maybe because it was declared and then written to?

Reply:
Yeah I am not sure why it didn't warning either, but I couldn't find a single other place where it was used in ICU code.